<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskWindow</class>
 <widget class="QMainWindow" name="taskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>511</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/background/resource/background/bg10.jpg)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createTaskButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
border:2px solid rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>创建任务</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="taskTab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>451</width>
      <height>461</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;mK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="createTabButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>590</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
border:2px solid rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>创建新的分类</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delTaskButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
border:2px solid rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTaskButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;
border:2px solid rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>开始任务</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
